### ISSUE TITLE
Browser discrepancies with PDOM combo box

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#476]
@twant and I discovered a few interesting differences between the molarity combo box on firefox and chrome. This was while working on phetsims/molarity#57. 

* We also noticed that on chrome the role of the button was `combobox`, but couldn't find a reasoning. In firefox the role is `pushbutton`.
* In chrome `aria-labelledby` attribute doesn't seem to fill in the accessible name as expected with both values (at least I thought that was what was expected). Instead we find that the second id in the aria-labelledby attribute ("itself") isn't being filled in properly. 

![screen shot 2019-02-18 at 7 39 45 pm](https://user-images.githubusercontent.com/6856943/52983183-5c68e500-3396-11e9-9d59-b957aca33134.png)

-------------------------------

### Next
This is further noticed when we deleted the first id from the attribute and found that the combo box had no accessible name: 
![screen shot 2019-02-18 at 7 31 58 pm](https://user-images.githubusercontent.com/6856943/52983215-83271b80-3396-11e9-91a9-1412013fa96d.png)

-------------------------------------------------------------------------------------

In firefox we don't really get specifics about how the accessible name is being filled in, though we know it is coming from aria-labelledby, but the accessible name is correctly filled in as the combination of both ids, so it looks like "Solute Drink mix":
![image](https://user-images.githubusercontent.com/6856943/52983311-08123500-3397-11e9-9da3-a4a32129f3d7.png)

Tagging @jessegreenberg and @terracoda to comment. Perhaps we have discussed this before, but I forget. Part of me thought that we knew about the broken name in chrome, but at the time we didn't support chrome. It would be good to bring this to the attention of the team now though, especially since we switched our JAWS  support from firefox to chrome.

**Comment by jessegreenberg** [comment]
We previously ran into accessible name discrepancies with Chrome in https://github.com/phetsims/a11y-research/issues/116.

I can confirm that what I hear with NVDA and JAWS matches what is shown by the dev tools. With NVDA in FF I hear
>Solute, Drink Mix, Button, Sub Menu

In JAWS in Chrome I hear:
> Solute button, menu

Unsure what the discrepancy is or what Chrome isn't happy with.  The second test test makes me wonder if Chrome doesn't like that aria-labelledby is self referential and points to the button itself. Removing aria-labelledby the accessible name "Drink Mix" shows up for the button:
![image](https://user-images.githubusercontent.com/6396244/53110368-c43f3d00-3508-11e9-9f94-02bdc67a61a4.png)

**Comment by terracoda** [comment]
@zepumph, sorry I missed this issue. 
The two `aria-labelledby` id's are applied correctly in Safari and VoiceOver.

Is there another way to dynamically rename the button somehow with javascript, perhaps using a single `aria-labelledby` or a single string for the button's accessible name?

This seems like a bug in Chrome. Is there an option to file a bug?

**Comment by zepumph** [comment]
>The two aria-labelledby id's are applied correctly in Safari and VoiceOver.

Can you please say what it looks like specifically. Is it the same as either item in https://github.com/phetsims/sun/issues/476#issuecomment-465670517. Tagging this issue because it may be important before Molarity is published. @terracoda do you think that we need to work on this for Molarity?

**Comment by terracoda** [comment]
Some screen readers say sub menu, some say pop-up button.
Closing this issue.
